openmetadata-ui/src/main/resources/ui/src/utils/TourUtils.tsx

```diff
@@ -240,7 +238,7 @@ export const getSteps = (value: string, clearSearchTerm: () => void) => {
           />
         </p>
       ),
-      selector: '#profilerDataQuality',
+      selector: '[data-testid="Profiler & Data Quality"]',
```

Comment:
tests id should be camelCase

Reply:
Yes it will be updated once all the entity will have the same Tab component, I will cover this in my last patch of tab cleanup